src/main/java/mate/academy/validation/password/FieldMatch.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.validation.password;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Constraint(validatedBy = FieldMatcher.class)
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface FieldMatch {
+    String field();
+
+    String fieldMatch();
```

Comment:
would it be better to:
```suggestion
    String[] fields();
```

and then:
```
@FieldMatch(fields = {"password", "repeatPassword"})
```

Reply:
refactored